pkg/kubelet/kubelet.go

```diff
@@ -1633,7 +1634,7 @@ func (kl *Kubelet) syncPod(o syncPodOptions) error {
 		return err
 	}
 	if err := kl.volumeManager.WaitForAttachAndMount(defaultedPod); err != nil {
-		kl.recorder.Eventf(pod, api.EventTypeWarning, events.FailedMountVolume, "Unable to mount volumes for pod %q: %v", format.Pod(pod), err)
```

Comment:
I propose we remove this event, it is not useful and the true FailedMount event will surface after it from the OperationExecutor, it just adds noise IMO.  I commented for now waiting for review